kubectl-rook-ceph.sh

```diff
@@ -95,6 +95,7 @@ function flag_no_value() {
 # When a non-flag arg is reached, stop parsing and return the remaining args in REMAINING_ARGS.
 REMAINING_ARGS=()
 function parse_flags() {
+  REMAINING_ARGS=()
```

Comment:
This changes the functionality of `parse_flags()` which affects all other commands. This was intentionally left out of the function so that it doesn't destroy the value of `REMAINING_ARGS` until it successfully ends.

Reply:
Let's take a case in which we don't pass the deployment name by mistake, and if we don't refresh the REMAINING_ARGS

`./kubectl.sh  debug start --alternate-image imag1`

If we use REMAINING_ARGS outside parse_flags() ,

It will update   Deployment_Name to `debug ` as `REMAINING_ARGS` still contains `debug start --alternate-image 2` when it was called at the very starting.

So whenever there is no non flag argument it will not update it


Maybe I can discuss this with you in a call

Reply:
Hmm... let me pull it and take a look. Maybe I have something mixed up.